packages/image_picker/ios/Classes/ImagePickerPlugin.m

```diff
@@ -263,7 +264,20 @@ - (void)imagePickerController:(UIImagePickerController *)picker
                            options:nil
                      resultHandler:^(NSData *_Nullable imageData, NSString *_Nullable dataUTI,
                                      UIImageOrientation orientation, NSDictionary *_Nullable info) {
-                       [weakSelf saveImageWithOriginalImageData:imageData image:image];
+                       FLTImagePickerMIMEType type = kFLTImagePickerMIMETypeDefault;
```

Comment:
we can move `mage = [FLTImagePickerImageUtil scaledImage:image maxWidth:maxWidth maxHeight:maxHeight];` inside the block just as we did for the gifs. Also, everything in the block might be able to move to a separate method like `[weakSelf saveImageWithOriginalImageData:data image:image type:type maxWidth:width maxHeight:height];`

Reply:
I move these to ```FLTImagePickerPhotoAssetUtil#saveImageWithOriginalImageData```.